src/component/Todo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import Nav from "./Nav";
+import TodoList from "./TodoList";
+import TodoModal from "./TodoModal";
+
+const Todo: React.FC = () => {
+  const [modalDisplay, setModalDisplay] = useState<boolean>(false);
+  const [prevValueOfTodo, setPrevValueOfTodo] = useState<string>("");
+  const [prevIdOfTodo, setPrevidOfTodo] = useState<number>(0);
+  const [prevTimeOfTodo, setPrevTimeOfTodo] = useState<Date>(new Date());
+
+  return (
+    <div className="container">
+      <Nav setModalDisplay={setModalDisplay} />
+      <TodoModal
+        prevValueOfTodo={prevValueOfTodo}
+        setPrevValueOfTodo={setPrevValueOfTodo}
+        modalDisplay={modalDisplay}
+        setModalDisplay={setModalDisplay}
+        prevIdOfTodo={prevIdOfTodo}
+        setPrevidOfTodo={setPrevidOfTodo}
+        prevTimeOfTodo={prevTimeOfTodo}
+        setPrevTimeOfTodo={setPrevTimeOfTodo}
+      />
+      <TodoList
+        setModalDisplay={setModalDisplay}
+        setPrevValueOfTodo={setPrevValueOfTodo}
+        setPrevidOfTodo={setPrevidOfTodo}
+        setPrevTimeOfTodo={setPrevTimeOfTodo}
```

Comment:
can you make it in a single data? you are creating 3 variables and 3 setters, you can do it via a single object. @Rishab-innovative 
What happens if ToDo has 3-4 more fields, you will be going to create 3-4 more variables and their setters.
which is not a good practice @Rishab-innovative